dbft.go

```diff
@@ -5,39 +5,39 @@ import (
 	"time"
 
 	"github.com/nspcc-dev/dbft/block"
+	"github.com/nspcc-dev/dbft/crypto"
 	"github.com/nspcc-dev/dbft/payload"
 	"github.com/nspcc-dev/dbft/timer"
-	"github.com/nspcc-dev/neo-go/pkg/util"
 	"go.uber.org/zap"
 )
 
 type (
 	// DBFT is a wrapper over Context containing service configuration and
 	// some other parameters not directly related to dBFT's state machine.
-	DBFT struct {
-		Context
-		Config
+	DBFT[H crypto.Hash, A crypto.Address] struct {
+		Context[H, A]
+		Config[H, A]
 
 		*sync.Mutex
-		cache      cache
+		cache      cache[H, A]
 		recovering bool
 	}
 
 	// Service is an interface for dBFT consensus.
-	Service interface {
+	Service[H crypto.Hash, A crypto.Address] interface {
```

Comment:
It this interface useful at all?

Reply:
Yes, it gives a hint to the user of dBFT what is expected from the dBFT service:
https://github.com/nspcc-dev/dbft/blob/92d9aece4e70265f0b59dad564a756a9da2a0b62/README.md?plain=1#L32-L38.